### PR TITLE
CLI release notes for 0.24.0

### PR DESCRIPTION
**Pull request by marians**
RFR @zyndiecate @ewoutp

### CODE REVIEW

File: content/reference/cli/release-notes.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ weight = 10
 
 Here we list the more important changes we made between releases. See our [installation reference](/reference/cli/installation/) for information on how to update to the latest CLI.
 
+## Version 0.24.0
+
+Released 2015-11-11
+
+* `swarm update` for a component should no longer exit with an EOF error. ([cli#366](https://github.com/giantswarm/cli/issues/366))
```
Comment (zyndiecate) [inline content/reference/cli/release-notes.md]: We should then just close the issue, shouldn't we?

File: content/reference/cli/release-notes.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 +++
 title = "Release Notes"
 description = "Release notes and changelog for the swarm CLI, showing you what has changed from release to release."
-date = "2015-10-30"
+date = "2015-11-10"
```
Comment (zyndiecate) [inline content/reference/cli/release-notes.md]: This is still a mystery to me why we need to update this manually. :)

File: content/reference/cli/release-notes.md (hunk 1 above)
Comment (marians) [inline content/reference/cli/release-notes.md]: Done

File: content/reference/cli/release-notes.md (hunk 2 above)
Comment (marians) [inline content/reference/cli/release-notes.md]: Because our world is full of imperfections, some small, some large.

**Pull request merged**